recipes/ledger/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -ex
+
+cmake . \
+  -DUSE_PYTHON=ON \
+  -DCMAKE_INSTALL_PREFIX="${PREFIX}"
```

Comment:
add `-DCMAKE_INSTALL_LIBDIR=lib` here.

Reply:
Could you explain this suggestion?

Reply:
this will install the libraries to `${PREFIX}/lib` instead of `${PREFIX}/lib64`